cid-redirects.json

```diff
@@ -259,6 +259,7 @@
   "/05Search/Search-Query-Language/Transaction-Analytics/Transactionize-operator": "/docs/search/search-query-language/transaction-analytics/transactionize-operator",
   "/05Search/Subqueries": "/docs/search/subqueries",
   "/05Search/Time-Compare": "/docs/search/time-compare",
+  "/07Sumo-Logic-Apps": "/docs/integrations/sumo-apps",
```

Comment:
/07Sumo-Logic-Apps/26Apps_for_Sumo

Reply:
Sadly not the actual request from Nick, but a nice addition.

Reply:
Can you confirm the request from Nick? 

/07Sumo-Logic-Apps = /docs/integrations

and

/07Sumo-Logic-Apps/26Apps_for_Sumo = /docs/integrations/sumo-apps